.github/workflows/build.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
       - name: Install modules.
         run: yarn install --network-timeout 600000 && yarn allow-scripts
+      - name: setup env file
+        run: echo "VITE_LD_ENVIRONMENT_ID=${{ secrets.VITE_LD_ENVIRONMENT_ID_TEST }}" > .env
```

Comment:
we could make this a custom action under `.github/workflows/setupLD/action.yml`

then here in this step we would just set
```
- name: Setup Launch Darkly
        uses: ./.github/workflows/setupLD
```
 
 This way we don't have to duplicate the same 
```
  run: echo "VITE_LD_ENVIRONMENT_ID=${{ secrets.VITE_LD_ENVIRONMENT_ID_TEST }}" > .env
```
  everywhere and if we need to modify it in the future or add another LD setup step, we can do it in one place

Reply:
We would need two custom actions since one is for testing and other for production.

Reply:
I tried it here but reverted in the end:
- I still needed to duplicate two lines of code on every action
- In some workflows the github cannot find the action
- After fixing the above, Github complains that actions has no access to secrets.

In the end, lost a lot of time for almost no benefit and reverted back to original.